scripts/get_luet.sh

```diff
@@ -15,10 +15,20 @@ LUET_CONFIG_PROTECT=${LUET_CONFIG_PROTECT:-1}
 LUET_PACKAGE="${LUET_PACKAGE:-toolchain/luet}"
 LUET_ARCH="${LUET_ARCH:-x86_64}"
 LUET_INSTALL_FROM_COS_REPO="${LUET_INSTALL_FROM_COS_REPO:-true}"
-if [[ "$LUET_ARCH" == "x86_64" ]]; then
-  LUET_ARCH="amd64" # stupid golang and their names for arches ¬_¬
+
+if [ -z "$LUET_ARCH" ]; then
+    LUET_ARCH=$(uname -m)
 fi
 
+case $LUET_ARCH in
+    amd64|x86_64)
+        LUET_ARCH=amd64
+        ;;
+    arm64|aarch64|arm*)
+        LUET_ARCH=armv7
```

Comment:
is this correct? armv7 is not aarch64...

Reply:
oh well, we need to as it is used in the repo generation :sweat_smile:

Reply:
well, as long ar the LUET_ARCH is not used for constructing repos for example, it should be ok. but the only repos we have are constructed with arm64 naming on there so...